Kickstarter-iOS/Views/Controllers/BetaToolsViewController.swift

```diff
@@ -14,7 +14,9 @@ internal final class BetaToolsViewController: UIViewController {
   @IBOutlet fileprivate weak var betaFeedbackButton: UIButton!
   @IBOutlet fileprivate weak var betaTitleLabel: UILabel!
   @IBOutlet fileprivate weak var languageSwitcher: UIButton!
+  @IBOutlet fileprivate weak var languageTitleLabel: UILabel!
```

Comment:
Since our `Beta Tools` use some of the `Settings`'s styles it was only fair to update the colors there as well